federation/ldap/src/main/java/org/keycloak/storage/ldap/mappers/UserAttributeLDAPStorageMapper.java

```diff
@@ -259,6 +269,12 @@ public void setFirstName(String firstName) {
                     super.setFirstName(firstName);
                 }
 
+                @Override
+                public void setEmailVerified(boolean verified) {
```

Comment:
I am seeing "setEmailVerified" overriden, but not "isEmailVerified" . Is this a mistake?

Reply:
No need to override "isEmailVerified". Even without this pull request read of emailVerified attribute is working fine, only write to LDAP had issues.

Reply:
AFAIK if you don't override "isEmailVerified", then it reads the value of emailVerified from the Keycloak DB. Which can be fine if it has same value as LDAP. However for example in the case like this:
- Keycloak user is registered with emailVerified "true"
- Later administrator directly update user in LDAP and set "emailVerified" to false
- Keycloak will now incorrectly read emailVerified "true" from the Keycloak DB without considering newly updated value from the LDAP